sbom.md

```diff
@@ -0,0 +1,86 @@
+# SBOM of HAMi
+
+We provide a Software Bill of Materials (SBOM) for this project, which includes a detailed list of all components, libraries, and dependencies used in the project. 
+This helps to ensure the transparency and security of the software by tracking and managing its dependencies.
+
+
+Software Bill of Materials:
+- [Project 'HAMi'](#project-hami)
+
+### Project 'HAMi'
+
+ | Name | Version | License | 
+ |---|---|---| 
+ | github.com/NVIDIA/go-gpuallocator | v0.3.2 | Apache-2.0 | 
```

Comment:
How to update the versions of these dependent components

Reply:
i used an open-source sbom tool to scan the hami project.